### PR TITLE
Remove React ionicons

### LINKED ISSUE #231: Fix EventCard ComponentWillMount Warning
When running tests or in the browser on `localhost:3000/events`, we see this warning:
image

Track down the issue (it is likely the Ionicon component [here](https://github.com/SCE-Development/Core-v4/blob/44e25e174076e97f9556acd5a2323c423d856e83/src/Pages/Events/EventCard.js#L30)) 

### The solution
If it is the Ionicon component, you can do one of two things:

1. Try to update the ionicon dependency [like this](https://stackoverflow.com/questions/43127863/node-update-a-specific-package)
2. Remove the Ionicon component.

If you decide to remove Ionicon component, render the icon as an svg and store the svg in [this file](https://github.com/SCE-Development/Core-v4/blob/44e25e174076e97f9556acd5a2323c423d856e83/src/Pages/Overview/SVG.js#L3).

### PR DESCRIPTION
**Pull request by VuongAlexander**
Resolves #231

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ is deemed necessary and appropriate to the circumstances.
 
 Project maintainers who do not follow or enforce the Code of Conduct in good
 faith may face temporary or permanent repercussions as determined by other
-members of the project's leadership.
+members of the project's leadership.
```
Comment (evanugarte) [inline CODE_OF_CONDUCT.md:56]: not sure why this line was changed, can you reset it to your dev branch? see [this answer on stackoverflow](https://stackoverflow.com/a/37972822). just change the branchname `master` to `dev`.

File: src/Pages/Events/EventCard.js [hunk 2]
```diff
@@ -28,10 +27,7 @@ function EventCard(props) {
             {getDateWithSlashes(eventDate.slice(0, 10))} {startTime} - {endTime}
           </Row>
           <Row className='event-title'>{title}</Row>
-          <Row className='event-location'>
-            <Ionicon icon='md-pin' fontSize='1.5rem' color='#414141bd' />
-            {eventLocation}
-          </Row>
+          <Row className='event-location'>{eventLocation}</Row>
```
Comment (evanugarte) [inline src/Pages/Events/EventCard.js:30]: import the function you created to show the map pin, and render it here like below
```suggestion
          <Row className='event-location'>
            {mapPinSymbol()}
            {eventLocation}
          </Row>
```

File: src/Pages/Events/EventInfoModal.js [hunk 3]
```diff
@@ -23,14 +22,12 @@ function EventInfoModal(props) {
         <Container className='center'>
           <Col>
             <Row className='event-date'>
-              <Ionicon icon='md-time' fontSize='1.5rem' color='#0098ab' />
               {getDateWithSlashes(currentEvent.eventDate.slice(0, 10))}{' '}
```
Comment (evanugarte) [inline src/Pages/Events/EventInfoModal.js:25]: import the function you created to show the clock, and render it here like below
```suggestion
          <Row className='event-location'>
            {clockSymbol()}
            {eventLocation}
          </Row>
```

Comment (VuongAlexander) [reply src/Pages/Events/EventInfoModal.js:25]: It says event-location is not defined

Comment (evanugarte) [reply src/Pages/Events/EventInfoModal.js:25]: what im saying is incorporate the functions you wrote

File: test/frontend/EventCard.test.js [hunk 4]
```diff
@@ -6,8 +6,9 @@ import { expect } from 'chai';
 
 import EventCard from '../../src/Pages/Events/EventCard';
 import Adapter from 'enzyme-adapter-react-16';
-import { Row } from 'reactstrap';
+import { Row, Col } from 'reactstrap';
```
Comment (evanugarte) [inline test/frontend/EventCard.test.js:9]: do you need `Col`?

**Pull request merged**